src/lib/db/client-applications-store.ts

```diff
@@ -35,8 +36,49 @@ const mapRow: (any) => IClientApplication = (row) => ({
     icon: row.icon,
     lastSeen: row.last_seen,
     announced: row.announced,
+    project: row.project,
+    environment: row.environment,
 });
 
+const reduceRows = (rows: any[]): IClientApplication[] => {
```

Comment:
Since we are joining `usage` table, there are duplicate entries, which we are reducing into `usage` object.